### ISSUE TITLE
Feature request: Public Profiles should have publicly accessible team builds

### ISSUE THREAD

**Issue by weiluntong** [PeteAndersen/swarfarm#75]
I think that if users want, they can set team builds they create to be publicly accessible. If they decide to do that, then the public teams should be pooled into a centrally located area within the site for anyone to access and even copy to their own profile to be modified to suit them personally.

**Comment by PeteAndersen** [comment]
This was a feature I had planned for a teams revamp I am working on. Teams will be associated with a specific dungeon and floor/difficulty, which will then be browseable via a dungeon section

**Comment by GuyonWoW** [comment]
Is this still under work?

**Comment by QuatZo** [comment]
No, it's not. Currently it's pretty low on my TODO list. So low, that I forgot about it.
Making teams public is not a bad idea tho. It's also quite fast to implement (except copying teams to your account), so if it's really needed/useful, I can move it higher on the list.

**Comment by GuyonWoW** [comment]
There are a few websites that does that, i just think having everything on one platform makes everything easier.
Sorry about flooding you with tickets tonight :P

**Comment by QuatZo** [comment]
I understand, I'll move it higher. Of course it will be a single checkbox saying "Show this team on my public page", at least for now. Whole thing with copying teams and connecting them to dungeon runs needs to wait. The earlier needs a lot of testing and validation, the later is pretty big change in Bestiary, Data Logs and Profile (which means in whole website).

> Sorry about flooding you with tickets tonight :P

All good. To be fair, the more the better.

**Comment by QuatZo** [comment]
I see that Teams are already publicly available, if your profile is set to public. There's no option to hide some team tho. As of today, I won't do any changes to it (unless you meant something else, then let me know), I'll just create TODO here for future reference.

- [ ] Create `private` Team for Public profile
- [ ] Connect Team (or TeamGroup) with DungeonLog
- [ ] Create `Copy` functionality for Team
- [ ] Add Most popular/Fastest Team in Dungeon Section

**Comment by GuyonWoW** [comment]
I wouldn't really mind about having all teams public, but that's only my 2 cent.
